src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/ApplicationPropertyPage.xaml.cs.xlf

```diff
@@ -42,19 +72,34 @@
         <target state="translated">Spouštěcí objekt</target>
         <note />
       </trans-unit>
-      <trans-unit id="DynamicEnumProperty|TargetFramework|Description">
-        <source>Specifies the version of .NET that the application targets. This option can have different values depending on which versions of .NET are installed on your computer.</source>
-        <target state="translated">Určuje verzi rozhraní .NET, na kterou aplikace cílí. Tato možnost může mít různé hodnoty v závislosti na tom, které verze rozhraní .NET jsou na počítači nainstalované.</target>
```

Comment:
These translations could also have been preserved. It's a shame our tooling doesn't do a better job of tracking this ID renames, in cases where the source strings are the same.

Reply:
I agree, it's weird how it picks up the changes and triggers localization updates.

Reply:
Was able to save these translations when I fixed some merge conflicts just now (with main).